substrate/frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1354,6 +1357,37 @@ pub mod pallet {
 	pub struct Pallet<T>(_);
 }
 
```

Comment:
If they have to be handled together, why not just store the whole struct?

Reply:
I think this is a good point it would save on storage writes. We're currently working on a multi block EPM pallet for the staking parachain and we'll try to improve there. I think for this PR it's useful to add the wrapper improvement and avoid migrations.

Reply:
They might need to be read separately and we might not want to read the whole snapshot when we need to read only the metadata.